.github/DISCUSSION_TEMPLATE/Help.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Discussion
+about: Describe a problem or seek help from the community
+
+---
+
+## Description
+<!-- A clear and concise description of the issue, and why you consider it to be a bug. -->
+
+## Your Environment
```

Comment:
For me, details regarding the environment usually are applicable for _Issues_ ticket.
Potentially, can we use _Additional Information_ section as you added below for _Feature request_?
So it could be something like:
```
## Additional information
<!-- Add any other context, environment information or screenshots -->
```